### DISCUSSION TITLE
How to mirror a git repository without the pull refs

### DISCUSSION THREAD

**Discussion by sylvainmouquet** [gitpython-developers/GitPython#1173] [Q&A] [answered]
Hello,

When i clone a GitHub project i get the list of pull request (git.reference).
I would like push only branches and tags but not pull request.
Example with git is present here : https://christoph.ruegg.name/blog/git-howto-mirror-a-github-repository-without-pull-refs.html

It's possible to do the same with gitpython

Thanks

**Comment by Byron** [comment, marked as answer]
The easiest way to go about this would probably be to use GitPython as easier way to [execute git commands](https://gitpython.readthedocs.io/en/stable/tutorial.html#using-git-directly). That way, git commands you know apply directly without having to think about how to do it with GitPythons own abstractions.

Even though I close this issue, please feel free to keep commenting for follow ups.
